src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/modelchecking/ModelCheckingStrategy.kt

```diff
@@ -61,7 +62,7 @@ internal class ModelCheckingStrategy(
     private var isReplayingSpinCycle = false
 
     // Tracker of objects' allocations and object graph topology.
-    override val objectTracker: ObjectTracker? = if (isInTraceDebuggerMode) null else LocalObjectManager()
```

Comment:
Does this mean we now track objects when in normal mode? And does that affect performance?

Reply:
Thanks for spotting this, there was an issue here. 

I enabled `LocalObjectManager` only for model checking mode, and for trace debugger mode use base object tracker. 

> Does this mean we now track objects when in normal mode? 

In trace debugger mode, we tracked them via `ObjectIdentityHashCodeTracker` and indirectly through global object numeration map.